src/main/java/yablunovskyi/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,27 @@
+package yablunovskyi.bookstore.dto;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.math.BigDecimal;
+import org.hibernate.validator.constraints.URL;
+
+public record BookDto(
```

Comment:
I'm not sure should we put validations on this class also, but let's leave this to mentors

Reply:
There is no sense to validate the response dto